content/thanks.md

```diff
@@ -0,0 +1 @@
+**Thanks for all the fish!! 🐟**
```

Comment:
Personal preference: Thank you for the fish!

Reply:
Bbbut this is the [official version](https://en.wikipedia.org/wiki/So_Long,_and_Thanks_for_All_the_Fish)!